frontend/app/assets/javascripts/spree/frontend/checkout/coupon-code.js.coffee

```diff
@@ -0,0 +1,34 @@
+Spree.onCouponCodeApply = (e) ->
+  coupon_code_field = $('#order_coupon_code')
+  coupon_code = $.trim(coupon_code_field.val())
+  return if coupon_code == ''
+
+  coupon_status = $("#coupon_status")
```

Comment:
I can swap them. I can also give it a go with the changelog entry. It must be the one [at the root](https://github.com/solidusio/solidus/blob/master/CHANGELOG.md#solidus-130-unreleased) of the project, right? - just to add a bullet point at the bottom of 1.3?